data/json/mutations/mutations.json

```diff
@@ -3598,6 +3598,34 @@
       "strength_damage": { "damage_type": "cut", "amount": 4 }
     }
   },
+  {
+    "type": "mutation",
+    "id": "TOE_CLAWS",
+    "name": { "str": "Clawed Feet" },
+    "copy-from": "RAP_TALONS",
+    "description": "Your toenails have grown long and sharp, preventing you from wearing shoes, but providing you with a set of powerful claws on your feet.  Fortunately they don't impede your movement.",
+    "category": [ "LIZARD" ],
+    "attacks": {
+      "attack_text_u": "You slash %s with your claws",
+      "attack_text_npc": "%1$s slashes %2$s with their claws",
+      "chance": 15,
+      "strength_damage": { "damage_type": "cut", "amount": 3 }
+    }
+  },
+  {
+    "type": "mutation",
+    "id": "FOOT_TALONS",
+    "name": { "str": "Taloned Feet" },
+    "copy-from": "RAP_TALONS",
+    "description": "Your feet have shifted into bird-like talons.  Instead of toes you now have three long talons at the front and one at the back, ending in wicked claws and preventing you from wearing shoes.  Fortunately they don't impede your movement.",
+    "category": [ "BIRD" ],
+    "attacks": {
```

Comment:
So I watched this video <https://www.youtube.com/watch?v=DtEoPuL1wF8>  and I think that maybe it would be best to remove the natural attack here and instead give bird's taloned feet a dash attack with a range of one square.  What do you think?

Reply:
If you went that direction it would give the bird talons a more dedicated role that makes more sense than randomly adding an extra attack when they've already got beak attacks.